### ISSUE TITLE
deep_speech_2/examples/tiny run_data.sh 运行时报错

### ISSUE THREAD

**Issue by JoeBlack220** [PaddlePaddle/models#522]
root@3328a2284d27:/DeepSpeech/examples/tiny# sh run_train.sh
-----------  Configuration Arguments -----------
augment_conf_path: conf/augmentation.config
batch_size: 16
dev_manifest: data/tiny/manifest.tiny
init_model_path: None
is_local: 1
learning_rate: 1e-05
max_duration: 27.0
mean_std_path: data/tiny/mean_std.npz
min_duration: 0.0
num_conv_layers: 2
num_iter_print: 100
num_passes: 20
num_proc_data: 1
num_rnn_layers: 3
output_model_dir: ./checkpoints/tiny
rnn_layer_size: 2048
share_rnn_weights: 1
shuffle_method: batch_shuffle_clipped
specgram_type: linear
test_off: 0
train_manifest: data/tiny/manifest.tiny
trainer_count: 8
use_gpu: 1
use_gru: 0
use_sortagrad: 1
vocab_path: data/tiny/vocab.txt
------------------------------------------------
I1206 11:44:56.762332    47 Util.cpp:166] commandline:  --use_gpu=1 --rnn_use_batch=True --log_clipping=True --trainer_count=8 
F1206 11:44:57.226635    47 hl_cuda_cublas.cc:137] Check failed: CUBLAS_STATUS_SUCCESS == g_cublasStat (0 vs. 1) Cublas Error: [cublas status]: not initialized [cublas init] Cublas create handle faild!
*** Check failure stack trace: ***
    @     0x7f7b10b7f04d  google::LogMessage::Fail()
    @     0x7f7b10b81398  google::LogMessage::SendToLog()
    @     0x7f7b10b7eb5b  google::LogMessage::Flush()
    @     0x7f7b10b8226e  google::LogMessageFatal::~LogMessageFatal()
    @     0x7f7b10b1d7a1  hl_cublas_init()
    @     0x7f7b10b2a6e4  hl_create_global_resources()
    @     0x7f7b10b2b1a4  hl_specify_devices_start()
    @     0x7f7b10b2b47d  hl_start()
    @     0x7f7b10aace7e  paddle::initMain()
    @     0x7f7b10b65621  initPaddle()
    @     0x7f7b10601347  _wrap_initPaddle
    @           0x4c468a  PyEval_EvalFrameEx
    @           0x4c9d8f  PyEval_EvalFrameEx
    @           0x4c2765  PyEval_EvalCodeEx
    @           0x4de6fe  (unknown)
    @           0x4b0cb3  PyObject_Call
    @           0x4c6ad1  PyEval_EvalFrameEx
    @           0x4c2765  PyEval_EvalCodeEx
    @           0x4ca8d1  PyEval_EvalFrameEx
    @           0x4c2765  PyEval_EvalCodeEx
    @           0x4ca8d1  PyEval_EvalFrameEx
    @           0x4c2765  PyEval_EvalCodeEx
    @           0x4c2509  PyEval_EvalCode
    @           0x4f1def  (unknown)
    @           0x4ec652  PyRun_FileExFlags
    @           0x4eae31  PyRun_SimpleFileExFlags
    @           0x49e14a  Py_Main
    @     0x7f7b35555830  __libc_start_main
    @           0x49d9d9  _start
    @              (nil)  (unknown)
Aborted (core dumped)
Fail in training!

**Comment by kuke** [comment]
@JoeBlack220 The DS2 project under this repo is no longer maintained, Please go to our new repo https://github.com/PaddlePaddle/DeepSpeech

**Comment by shanyi15** [comment]
您好，此issue在近一个月内暂无更新，我们将于今天内关闭。若在关闭后您仍需跟进提问，可重新开启此问题，我们将在24小时内回复您。因关闭带来的不便我们深表歉意，请您谅解~感谢您对PaddlePaddle的支持! 
 Hello, this issue has not been updated in the past month. We will close it today for the sake of other user‘s experience. If you still need to follow up on this question after closing, please feel free to reopen it. In that case, we will get back to you within 24 hours. We apologize for the inconvenience caused by the closure and thank you so much for your support of PaddlePaddle Group!
